addon/components/o-s-s/information-section.hbs

```diff
@@ -0,0 +1,41 @@
+<div
+  data-control-name="information-section"
+  class="oss--information-section border-radius-md border-color-gray-200 box-shadow-md"
```

Comment:
```suggestion
  class="oss-information-section border-radius-md border-color-gray-200 box-shadow-md"
```
According to BEM `--` is to modifier, maybe just use `-`

Reply:
Cleaned up